build/build.xml

```diff
@@ -241,6 +241,79 @@
 		</sequential>
 	</macrodef>
 
+	<macrodef name="build-addon-without-help-indexes" description="build the specified addon">
```

Comment:
It would be better (for maintenance) if the common "code" were moved to a base macrodef to be used by both "build-addon" macrodefs.
But, since we are moving away from Ant build that shouldn't be a problem.